### ISSUE TITLE
Antenna and source tables in cal_solutions have repeated values

### ISSUE THREAD

**Issue by mooneyse** [lofar-astron/prefactor#241]
This isn't an issue which causes the pipeline to crash so it is low priority. But inspecting the `cal_solutions.h5` resulting from `Pre-Facet-Calibrator.parset`, the antenna table has 4 copies of each antenna and the source table has 4 copies of the pointing. e.g. Here is the source table in the calibrator solset:

![image](https://user-images.githubusercontent.com/20988307/61144237-42a67680-a4cc-11e9-95ef-c25f01dece1f.png)

Since these aren't actually used for anything, it isn't a big deal. Just thought it was worth noting.

**Comment by tikk3r** [comment]
It being each effect would make sense, as different soltabs can have different antennas, but someone else may know for sure.

**Comment by mooneyse** [comment]
Well each tab will have its own antenna axis anyway, so I guess the best thing would be for the antenna table to have a list of all antennas that appear in any of the soltab axes.

**Comment by revoltek** [comment]
This is how the h5parm standard works: each solset has 2 tabs, one with antenna and one with directions that are used to lookup coordinates if needed. If an antenna/direction is in one axis, then it should be also in this table. It can happen that then antenna table and an axis have different entries (e.g. if you solve only for a subset of data).

**Comment by mooneyse** [comment]
Yup, understood. I just wouldn't expect the antenna table to have multiple entries for each antenna. Initially when I saw the antenna table had ~345 antennas in it, I figured something had gone wrong, but it is just the way they are appended in the pipeline. Ideally there would be a final step in Prefactor to remove duplicates, but it is low priority admittedly.
